Lesson33/Gokhan/src/components/UserDetailsForm/index.jsx

```diff
@@ -0,0 +1,33 @@
+import './styles.css';
+import { useContext, useState } from 'react';
+import { UserContex } from '../../context/UserContext';
```

Comment:
I cannot see the UserContext file added in this application. You need to first create the file with Context and Provider component to be able to use it.